src/com/facebook/buck/android/AndroidLibraryDescription.java

```diff
@@ -47,17 +52,20 @@
 import java.nio.file.Path;
 
 public class AndroidLibraryDescription
-    implements Description<AndroidLibraryDescription.Arg>, Flavored {
+    implements Description<AndroidLibraryDescription.Arg>, Flavored,
+       ImplicitDepsInferringDescription<AndroidLibraryDescription.Arg> {
 
   public static final BuildRuleType TYPE = BuildRuleType.of("android_library");
 
   private static final Flavor DUMMY_R_DOT_JAVA_FLAVOR =
       AndroidLibraryGraphEnhancer.DUMMY_R_DOT_JAVA_FLAVOR;
 
   private final JavacOptions defaultOptions;
+  private final ScalaBuckConfig scalaBuckConfig;
 
-  public AndroidLibraryDescription(JavacOptions defaultOptions) {
+  public AndroidLibraryDescription(JavacOptions defaultOptions, ScalaBuckConfig scalaConfig) {
```

Comment:
Not sure how to improve that, we could introduce some additional class for this, something like `JvmLanguagesConfig`, but it's just moving the same things from arguments list to fields. Any better suggestions?